thumbs/uploader.go

```diff
@@ -11,7 +10,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
-	"github.com/prometheus/common/log"
+	log "github.com/sirupsen/logrus"
```

Comment:
huh good catch!

Reply:
Yeah this cost me so much time on travis over the weekend. There was no error in go mods ofcoarse so I had no idea what package was causing the problem for building. Eventually I was doing some trial and error and found a link online that lead me to try this package.